### DISCUSSION TITLE
Empty instace on AWS

### DISCUSSION THREAD

**Discussion by sebaslml** [Security-Onion-Solutions/securityonion#5985] [Unsupported Versions] [answered]
Hi, 
I noticed that if you cancel your intallation, or happens any error during intallation, when you go back to run your Security Onion 2 instance on aws, does not show the main installation menu.
![instace](https://user-images.githubusercontent.com/92940863/138505339-047be16c-4bec-45b9-8af6-3ca5493f5ff6.PNG)
It is totally empty! I can not use any command to start the security onion service.

Just the firts time I run this instace it show me this
![Main menu](https://user-images.githubusercontent.com/92940863/138505547-13a246ef-c14b-4c75-b9df-c0280095fb69.PNG)

What is going on? Every time this happens I need to lauch a new instance of Security onion 2 again.

**Comment by dougburks** [comment, marked as answer]
You should be able to do the following as shown at https://docs.securityonion.net/en/2.3/cloud-ami.html#prepare-nodes-with-ephemeral-storage:
```
cd /securityonion
sudo ./so-network-setup
```

**Comment by sebaslml** [comment]
THANKS A LOT!!!
IT WORKS!
